api/src/filters/all-exceptions.exception.filter.spec.ts

```diff
@@ -1,65 +1,66 @@
 import { AllExceptionsFilter } from './all-exceptions.exception.filter';
-import { HttpException, ArgumentsHost, HttpStatus } from '@nestjs/common';
+import { ArgumentsHost, NotFoundException } from '@nestjs/common';
```

Comment:
what happened to this test file?! 😅

Reply:
@hotzevzl it mainly received some heavy lifting:
1) changed it to actually use NestJS ecosystem (i.e. is injected in DI instead manually constructed)
2) updated some assertions to be reliable (`expect(responseMock.json).toHaveBeenCalled();`, `expect(new AllExceptionsFilter()).toBeDefined();`)

Sorry, couldn't resist :(